Sources/PackageModel/Product.swift

```diff
@@ -131,23 +137,22 @@ public enum ProductFilter: Equatable, Hashable {
         self = self.union(other)
     }
 
-    public func contains(_ product: String) -> Bool {
+    public func contains(_ product: String, isCommandPlugin: (String) -> Bool) -> Bool {
```

Comment:
That makes sense.  Because SwiftPM doesn't yet have a versioned API I suspect that any clients could adapt, but passing in the closure makes sense too.